### PR TITLE
feat: log loading of credentials from ssm and assuming roles

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: topo_processor/util/aws_credentials.py [hunk 1]
```diff
@@ -36,11 +37,13 @@ def init_roles() -> None:
     if aws_profile is None:
         return
 
+    get_log().debug("load_bucket_config", ssm=linz_ssm_bucket_config_name)
```
Comment (paulfouquet) [inline topo_processor/util/aws_credentials.py:40]: Is there a reason for you writing log messages with underscores `_`?

**Review by paulfouquet — APPROVED** [review]
:+1:

File: topo_processor/util/aws_credentials.py (hunk 1 above)
Comment (paulfouquet) [reply topo_processor/util/aws_credentials.py:40]: `s3_download` is a reference to the function `s3_download()`. Not a big deal, I was just curious :)

Comment (blacha) [reply topo_processor/util/aws_credentials.py:40]: Most of my other libs don't use spaces primarily so I can grep for them without quoting it, I would often use `TileCase`

**Pull request merged**